diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/StringTemplateRuleWarnTest.kt

```diff
@@ -22,6 +21,7 @@ class StringTemplateRuleWarnTest : LintTestBase(::StringTemplateFormatRule) {
                 """
                     |class Some { 
                     |   val template = "${'$'}{::String} ${'$'}{asd.moo()}"
+                    |   val some = "${'$'}{foo as Foo}"
```

Comment:
Do you have tests that binary expressions in templates are considered valid (like `${foo as Foo}` instead of `${'$'}{..}`)?